data-node/pom.xml

```diff
@@ -412,6 +412,11 @@
             <version>2.6.0</version>
         </dependency>
 
+        <dependency>
+            <groupId>com.fasterxml.jackson.dataformat</groupId>
+            <artifactId>jackson-dataformat-smile</artifactId>
+            <version>2.8.9</version>
```

Comment:
Please don't specify a version here so we use the managed version from the parent pom.xml file.

Reply:
Removed, thanks!